beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classic:
```

Comment:
Probably, I am not a native speaker. "classical" (Built by 71,025 people (Used 574,814 times)) seems more popular than "classic" (Built by 26,016 people (Used 115,887 times)) on last.fm.